lib/get-dialect-ids.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from "vitest";
+import { getDialectIds, loadDialect } from "./keywords.js";
+import "../draft-2020-12";
+import "../draft-2019-09";
```

Comment:
Let's include all supported dialects to make sure we get only what we expect. (That should include the "stable" and "openapi" dialects.)